### PR TITLE
[AAE-10450] Stories for components in login module

### PR DESCRIPTION
**Pull request by DudaRobert**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [x] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

[AAE-10450](https://alfresco.atlassian.net/browse/AAE-10450)

**What is the new behaviour?**

Added stories

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/src/lib/login/components/login-dialog.component.html [hunk 1]
```diff
@@ -1,25 +1,23 @@
-<header
-    mat-dialog-title
-    data-automation-id="login-dialog-title">{{data?.title}}
+<header mat-dialog-title data-automation-id="login-dialog-title">
```
Comment (Qetikv) [inline lib/core/src/lib/login/components/login-dialog.component.html:1]: In my opinion, the styling that was before is better for readability

File: lib/core/src/lib/login/components/login-dialog.component.html [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
+        mat-button
         class="choose-action"
         data-automation-id="login-dialog-actions-perform"
         [disabled]="!isFormValid()"
-        (click)="submitForm()">{{ buttonActionName | translate}}
+        (click)="submitForm()">
+            {{ buttonActionName | translate}}
     </button>
-</mat-dialog-actions>
+</mat-dialog-actions>
```
Comment (Qetikv) [inline lib/core/src/lib/login/components/login-dialog.component.html:23]: You will need to add a new line at the end of the file

File: lib/core/src/lib/login/components/login-dialog.component.html (hunk 1 above)
Comment (DudaRobert) [reply lib/core/src/lib/login/components/login-dialog.component.html:1]: fixed in new commit

File: lib/core/src/lib/login/components/login-dialog.component.html (hunk 2 above)
Comment (DudaRobert) [reply lib/core/src/lib/login/components/login-dialog.component.html:23]: fixed in new commit

File: lib/core/src/lib/login/components/login-dialog.stories.component.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    selector: 'adf-login-dialog-storybook',
+    template: `<button mat-raised-button (click)="openLoginDialog()">
+        Open dialog
+    </button>`
+})
+export class LoginDialogStorybookComponent {
+
+    @Output() executeSubmit = new EventEmitter<any>();
+    @Output() error = new EventEmitter<any>();
+    @Output() closed = new EventEmitter<any>();
```
Comment (BSekula) [inline lib/core/src/lib/login/components/login-dialog.stories.component.ts:34]: can we add types here?

File: lib/core/src/lib/login/components/login-dialog.stories.component.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            this.executeSubmit.emit('executeSubmit');
+        },
+            (error) => {
+                this.error.emit(error);
+            },
+            () => {
+                this.closed.emit('closed');
+                this.dialog.closeAll();
+            });
+    }
```
Comment (BSekula) [inline lib/core/src/lib/login/components/login-dialog.stories.component.ts:63]: Wrong indentation

File: lib/core/src/lib/login/components/login.component.html [hunk 5]
```diff
@@ -159,4 +212,4 @@
 
         </div>
     </div>
-</div>
+</div>
```
Comment (BSekula) [inline lib/core/src/lib/login/components/login.component.html:215]: Missing end line

File: lib/core/src/lib/login/components/login-dialog.component.stories.ts [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+        docs: {
+            description: {
+                component: `Allows a user to perform a login via a dialog.`
+            }
+        }
+    },
+    argTypes: {
+        storybookInfo: {
+            control: 'none',
+            description: 'Use `fake-username` and `fake-password` to test functionality.',
```
Comment (BSekula) [inline lib/core/src/lib/login/components/login-dialog.component.stories.ts:48]: Do we need to add info about the `fake-username-CORS-error`?

**Review by BSekula — CHANGES_REQUESTED** [review]

File: lib/core/src/lib/login/components/login-dialog.component.stories.ts (hunk 6 above)
Comment (DudaRobert) [reply lib/core/src/lib/login/components/login-dialog.component.stories.ts]: Added in new commit

File: lib/core/src/lib/login/components/login.component.html (hunk 5 above)
Comment (DudaRobert) [reply lib/core/src/lib/login/components/login.component.html]: Added in new commit

File: lib/core/src/lib/login/components/login-dialog.stories.component.ts (hunk 4 above)
Comment (DudaRobert) [reply lib/core/src/lib/login/components/login-dialog.stories.component.ts:66]: fixed in new commit

File: lib/core/src/lib/login/components/login-dialog.stories.component.ts [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+        const data: LoginDialogComponentData = {
+            title: 'Perform a Login',
+            actionName: 'LOGIN',
+            logged: new Subject<any>()
+        };
+
+        this.dialog.open(
+            LoginDialogComponent,
+            {
+                data, panelClass: 'adf-login-dialog',
```
[the remark this answers is not in the document]
Comment (DudaRobert) [reply lib/core/src/lib/login/components/login-dialog.stories.component.ts]: fixed in new commit

File: lib/core/src/lib/login/components/login-dialog.component.html [hunk 8]
```diff
@@ -1,25 +1,25 @@
...
     </adf-login-dialog-panel>
 </mat-dialog-content>
 
 <mat-dialog-actions align="end">
-    <button
-        mat-button
-        (click)="close()"
-        data-automation-id="login-dialog-actions-cancel">{{ 'LOGIN.DIALOG.CANCEL' | translate }}
+    <button mat-button (click)="close()" data-automation-id="login-dialog-actions-cancel">
+        {{ 'LOGIN.DIALOG.CANCEL' | translate }}
```
[the remark this answers is not in the document]
Comment (DudaRobert) [reply lib/core/src/lib/login/components/login-dialog.component.html]: fixed in new commit :)

**Review by BSekula — APPROVED** [review]

**Pull request merged**